schemesVoteParams4Production/FundingRequestParams.json.txt

```diff
@@ -0,0 +1,14 @@
+{ // settings for funding request
+  "boostedVotePeriodLimit": 518400, // is 6 days
+  "daoBountyConst": 1, // absolutely minimal dowstake (would be nice if we could turn this off)
+  "minimumDaoBounty": 1, // (the contract does not allow 0)
+  "queuedVotePeriodLimit": 100800, // the proposal expires in 4 weeks
+  "queuedVoteRequiredPercentage": 51,
+  "preBoostedVotePeriodLimit": 0, // no preboosted state at all
```

Comment:
so no shift back to the q chance if proposal is boosted.

Reply:
yes, this basically disables a part of the protocol. 
It makes eveyrthing  easier from ux perspective, but I am not sure if we can really do this..

Reply:
ok, I will change this to a non-zero value :-)